helm/cluster-openstack/ci/ci-values.yaml

```diff
@@ -7,7 +7,12 @@ dnsNameservers:
 - 1.1.1.1
 - 8.8.8.8
 externalNetworkID: external-network-id
-kubernetesVersion: 1.20.9
+kubernetesVersion: v1.22.9
+kubernetesVersionImages:
```

Comment:
Is the customer going to set this? If not, can we hide this from the user? Why don't we just append `ubuntu-2004-kube-`?

Reply:
No, it is our default value for this specific cluster OpenStack version. Customer can always override it in other levels (GitOps layers)

> Why don't we just append ubuntu-2004-kube-?

the convention can change and then it fails